### ISSUE TITLE
The function pickRandom has no name

### ISSUE THREAD

**Issue by HK-SHAO** [josdejong/mathjs#2174]
![image](https://user-images.githubusercontent.com/24982550/114389982-45b22780-9bc8-11eb-88f8-6409d5f73bc0.png)
```javascript

math.evaluate('gamma').name
"gamma"
math.evaluate('gamma').name
"gamma"
math.evaluate('diag').name
"diag"
math.evaluate('random').name
"random"
math.evaluate('pickRandom').name
""
math.evaluate('xor').name
"xor"
```

**Comment by HK-SHAO** [comment]
`mathjs/src/function/probability/pickRandom.js/createPickRandom/`
A name argument is missing here.
```javascript

return typed({
    'Array | Matrix': function (possibles) {
      return _pickRandom(possibles, {})
    },

...
```
It should be like this.
```javascript
  return typed(name, {
    'Array | Matrix': function (possibles) {
      return _pickRandom(possibles, {})
    },

    'Array | Matrix, Object': function (possibles, options) {
      return _pickRandom(possibles, options)
    },
```
and ... How can I get involved in this repo?

**Comment by HK-SHAO** [comment]
![image](https://user-images.githubusercontent.com/24982550/114500954-a179c080-9c5b-11eb-8d15-fea0ac3089ba.png)
I don't know how to push 🤣

**Comment by josdejong** [comment]
Fixed via #2181, thanks @HK-SHAO

**Comment by josdejong** [comment]
Published now in `v9.4.0`
